posthog/utils.py

```diff
@@ -188,10 +188,12 @@ def get_git_commit() -> Optional[str]:
 
 
 def render_template(template_name: str, request: HttpRequest, context: Dict = {}) -> HttpResponse:
+    from loginas.utils import is_impersonated_session
```

Comment:
Any reason to not import this at the top of the file?

Reply:
The `utils` module from `loginas` has some imports related to the Django user model which is not available when our PostHog `utils` is loaded (we get an `AppRegistryNotReady` exception)

Reply:
Think this might be the culprit, `from django.contrib.auth.models import update_last_login`